server/datastore/mysql/migrations/tables/20251215205835_ConvertIdpToMdmIdpAccounts.go

```diff
@@ -0,0 +1,24 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20251215205835, Down_20251215205835)
+}
+
+func Up_20251215205835(tx *sql.Tx) error {
+	// https://github.com/fleetdm/fleet/issues/37168
+	_, err := tx.Exec(`UPDATE host_emails SET source = 'mdm_idp_accounts' WHERE source = 'idp'`)
```

Comment:
Am guessing that the `updated_at` updated to `CURRENT_TIMESTAMP` should not cause issues.